server/app.py

```diff
@@ -0,0 +1,17 @@
+# server.py
+from flask import Flask, render_template, request, url_for
+import speak
+
+app = Flask(__name__, static_folder="../static/dist", template_folder="../static")
+@app.route("/", methods=["GET","POST"])
+def index():
+    return render_template("index.html")
+
+@app.route("/hello/<text>/<lang>")
+def hello(text,lang):
+    if request.method == "GET":
+        speak.get_speak(text,lang)
+        return url_for("static", filename="mp3_path")
```

Comment:
`speak.get_speak()` will return the audio file name we've created, so we should assign that to a variable and pass to the front-end. **Example**:

```python
mp3_path = speak.get_speak(text, lang)
return url_for("static", filename=mp3_path)
```

Reply:
Done ! Thanks a lot !
Changed another small thing, the name of the file became a lot simpler, instead of generating a random number and concatenate it with filename, i found a python module to generate random strings, it's much simpler this way.

Reply:
Nice one, `uuid` is the right way go!